src/pytest_mock_resources/fixture/database/relational/postgresql.py

```diff
@@ -29,31 +45,25 @@ def create_postgres_fixture(*ordered_actions, **kwargs):
 
     session = kwargs.pop("session", None)
 
+    async_ = kwargs.pop("async_", False)
+
     if len(kwargs):
         raise KeyError("Unsupported Arguments: {}".format(kwargs))
 
     @pytest.fixture(scope=scope)
-    def _(_postgres_container, pmr_postgres_config):
-        database_name = _create_clean_database(pmr_postgres_config)
-        engine = get_sqlalchemy_engine(pmr_postgres_config, database_name)
-
-        assign_fixture_credentials(
-            engine,
-            drivername="postgresql+psycopg2",
-            host=pmr_postgres_config.host,
-            port=pmr_postgres_config.port,
-            database=database_name,
-            username=pmr_postgres_config.username,
-            password=pmr_postgres_config.password,
-        )
-
-        engine_manager = EngineManager(
-            engine, ordered_actions, tables=tables, default_schema="public"
-        )
-        for engine in engine_manager.manage(session=session):
-            yield engine
+    def _sync(_postgres_container, pmr_postgres_config):
+        engine_manager = create_engine_manager(pmr_postgres_config, ordered_actions, tables)
+        yield from engine_manager.manage_sync(session=session)
 
-    return _
+    @pytest.fixture(scope=scope)
+    def _async(_postgres_container, pmr_postgres_config):
```

Comment:
if you go with my above suggestions re making `manage_async` async, then this will need to be `async`, and i think that's fine because `pytest-asyncio` says that it handles async fixtures just fine.